src/gcp_scanner/utils.py

```diff
@@ -0,0 +1,166 @@
+# Copyright 2022 Google LLC
```

Comment:
It is ok to have it like that in this PR but in the future please try to separate such changes between independent PRs.

Reply:
Okay :)
Should I revert it?